### PR TITLE
Upgrade nginx to >= 12.0

### PR DESCRIPTION
**Pull request by daviddjh**
nginx cookbook removed its dependency on deprecated ohai cookbook at 12.0

### CODE REVIEW

**Comment by ramereth** [comment]
@daviddjh can you please address the cookstyle issues?

**Comment by daviddjh** [comment]
Sorry, pushed prematurely, fixing now

File: recipes/logrotate.rb [hunk 1]
```diff
@@ -16,10 +16,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
 include_recipe 'logrotate'
 
+Chef::DSL::Recipe.include(Nginx::Cookbook::Helpers)
+
+log_dir  = nginx_log_dir
+pid_file = nginx_pid_file
```
Comment (ramereth) [inline recipes/logrotate.rb:25]: Why can't these be used directly?

Comment (daviddjh) [reply recipes/logrotate.rb:25]: @ramereth When used in the custom resource, It leads to an undefined method error. Not sure exactly why, seems like some kind of scoping mismatch. Might be related to [this commit](https://github.com/sous-chefs/nginx/commit/0c9a3a445c4db3269ce878e15d350266c52d45d1).

Comment (ramereth) [reply recipes/logrotate.rb:25]: If you change it to use `Chef::DSL::Universal.include(Nginx::Cookbook::Helpers)`, it seems to work and pass cookstyle.

**Review by ramereth — APPROVED** [review]
Please do a minor bump.

**Comment by daviddjh** [comment]
!bump minor ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a minor-level version bump to v6.2.0.  Have a nice day!

**Pull request merged**